apis/core/lib/pipeline/trigger.ts

```diff
@@ -37,24 +43,39 @@ export const gitlab = trigger(job => {
   const form = new URLSearchParams({
     token: env.PIPELINE_TOKEN,
     ref: 'master',
-    'variables[JOB]': job.value,
     'variables[ENV]': env.PIPELINE_ENV,
   })
 
+  if (job.has(rdf.type, cc.TransformJob).values.length > 0) {
+    form.append('variables[TRANSFORM_JOB]', job.value)
+  }
+  if (job.has(rdf.type, cc.PublishJob).values.length > 0) {
+    form.append('variables[PUBLISH_JOB]', job.value)
+  }
+
   return {
     method: 'POST',
     body: form,
   }
 })
 
 export const github = trigger((job, params) => {
-  const body = {
-    ref: 'master',
-    inputs: {
-      job: job.value,
-    },
+  let body
+  if (job.has(rdf.type, cc.TransformJob).values.length > 0) {
+    body = {
+      ref: 'master',
+      inputs: {
+        transform_job: job.value,
+      },
+    }
+  } else {
+    body = {
+      ref: 'master',
+      inputs: {
+        publish_job: job.value,
+      },
+    }
```

Comment:
Slightly more readable IMO

```suggestion
  const jobParam = job.has(rdf.type, cc.TransformJob).values.length > 0 ? 'transform_job' : 'publish_job'

    body = {
      ref: 'master',
      inputs: {
        [jobParam]: job.value,
      }
    }
```

Reply:
:1st_place_medal: